.babelrc

```diff
@@ -6,5 +6,33 @@
   ],
   "plugins": [
     "lodash"
-  ]
+  ],
+  "env": {
+    "development": {
+      "plugins": [
+        ["react-transform", {
+          "transforms": [{
+              "transform": "react-transform-hmr",
+              "imports": [
+                "react"
+              ],
+              "locals": [
+                "module"
+              ]
+            }, {
+              "transform": "react-transform-catch-errors",
+              "imports": [
+                "react",
+                "redbox-react"
+              ]
+            }]
+        }]
+      ]
+    },
+    "test": {
+      "plugins": [
+        ["__coverage__", { "ignore": "test/" }]
```

Comment:
Since [isparta is no longer maintained](https://github.com/douglasduteil/isparta), we use the new [`babel-plugin-__coverage__`](https://github.com/dtinth/babel-plugin-__coverage__).  This plugin instruments the code only.  We use the standard [`karma-coverage`](https://github.com/karma-runner/karma-coverage) karma plugin to generate the coverage reports, but not for instrumenting.